core/src/main/java/org/apache/carbondata/core/statusmanager/FileFormat.java

```diff
@@ -46,6 +46,9 @@ public int ordinal() {
   }
 
   public static FileFormat getByOrdinal(int ordinal) {
+    if (ordinal < 0) {
```

Comment:
It should be invalid parameter, suggest to throw exception.

Reply:
OK